elastic-agent-client.proto

```diff
@@ -177,6 +178,78 @@ enum UnitType {
     OUTPUT = 1;
 }
 
+// Log level for the unit.
+enum UnitLogLevel {
+    TRACE = 0;
```

Comment:
This will be the default trace level if this field is left unset, should you reverse the order to be safe?

Reply:
It already doesn't exactly match the levels in beats https://pkg.go.dev/github.com/elastic/elastic-agent-libs@v0.2.9/logp#Level

Reply:
I was basing it off of zerolog but off by 1. Allowing it to be the `log_level - 1` to get the log level for zerolog. Why zerolog? Because zerolog is what we use in `fleet-server` and what we want to transition the Elastic Agent to using as well.

I think the numbers don't really matter. Would be happy to change it, maybe even place 0 as error. So that the default if not present is only error.

Reply:
I like having the default of 0 map to error, to avoid accidental log spam in case this is ever omitted.

Reply:
I agree. I am flipping this enum to make error 0.